src/app/views/authentication/profile/Profile.tsx

```diff
@@ -36,6 +36,13 @@ export class Profile extends Component<IProfileProps, IProfileState> {
       })
       : null;
 
+    const betaJsonUserInfo = actions
+      ? await actions.getProfileInfo({
+        selectedVerb: 'GET',
+        sampleUrl: BETA_USER_INFO_URL
```

Comment:
Calling the Beta Profile endpoint brings in a lot of new data. Consider manipulating the data received to fit in with the IUser type and update this to carry the extra data that is needed. It seems that the object change to the profile contents stored in state breaks the `user by email` sample query.

Reply:
https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/1008/commits/d492b75ff8a10ba86b4dba7e86ff032a355cd2b0 should fix the problem of overwriting the profile contents in state.